spec/formats/news_story_spec.rb

```diff
@@ -39,7 +39,7 @@ def and_i_fill_in_the_form_fields
   def and_i_add_some_tags
     stub_publishing_api_has_links(role_appointment_links)
 
-    expect(Document.last.document_type.tags.count).to eq(5)
+    expect(Document.last.current_edition.document_type.tags.count).to eq(5)
```

Comment:
Same comment about Document could be replaced with Edition. A good thing to sort here would be that we look up the first Document earlier in this test and last one here. These are predicated on the expectation that this is the only document in the db so we should probably stick with one. I'd suggest using `#last`.